docs/src/api.ts

```diff
@@ -0,0 +1,46 @@
+interface RequestObject {
+  method: string;
+  headers: {
+    'Content-Type': string;
+  };
+  body: string;
+}
+
+export function buildRequestSceneObject(method: string, scene: object): RequestObject {
+  return {
+    method: method,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ scene: scene }),
+  };
+}
+
+export async function handleRenderImageRequest(
```

Comment:
handle would be a good name on server side, sendRenderImageRequest or just renderImage would be better